src/components/BannerNotification.stories.tsx

```diff
@@ -0,0 +1,190 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { useState, SyntheticEvent } from 'react';
+import {
+  Alert as MuiAlert,
+  AlertTitle as MuiAlertTitle,
+  Box as MuiBox,
+  Button as MuiButton,
+  IconButton as MuiIconButton,
+  Snackbar as MuiSnackbar,
+  SnackbarOrigin as MuiSnackbarOrigin,
+} from '@mui/material';
+import {
+  CancelOutlined as MuiErrorIcon,
+  Check as MuiCheckIcon,
+  Close as MuiCloseIcon,
+  InfoOutlined as MuiInfoIcon,
+  WarningAmberOutlined as MuiWarningIcon,
+} from '@mui/icons-material'
+
+const meta: Meta<typeof MuiAlert> = {
+  title: 'Example/Banner Notification',
+  component: MuiAlert,
+  tags: ['autodocs'],
+  argTypes: {
+    children: {
+      control: 'text',
+      description: 'The content of the component.'
+    },
+    severity: {
+      control: 'select',
+      options: ['error', 'info', 'success', 'warning'],
+      description: 'The severity of the alert. This defines the color and icon used.'
+    },
+    title: {
+      control: 'text',
+      description: 'The title of the component.'
+    }
+  },
+}
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+/**
+ * Default banner notification story.
+ */
+export const Default: Story = {
+  args: {
+    severity: 'info',
+    children: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+    title: 'Heading',
+  },
+  render: (args) => {
+    let icon = null;
+
+    switch (args.severity) {
+      case 'error':
+        icon = <MuiErrorIcon fontSize="inherit" />;
+        break;
+      case 'warning':
+        icon = <MuiWarningIcon fontSize="inherit" />;
+        break;
+      case 'info':
+        icon = <MuiInfoIcon fontSize="inherit" />;
+        break;
+      case 'success':
+        icon = <MuiCheckIcon fontSize="inherit" />;
+        break;
+      default:
+        break;
+    }
+
+    const [, setOpen] = useState(false);
+
+    const handleClose = (event: SyntheticEvent | Event, reason?: string) => {
+      if (reason === 'clickaway') {
+        return;
+      }
+
+      setOpen(false);
+    };
+
+    return (
+      <MuiBox maxWidth="360px">
```

Comment:
@vinzkm check Mui native's component SnackBar.
it's has closest behavior to a banner notification with vital native props like anchorOrigin, onClose, close that represents actual behavior of a banner notification.
if you can use this as wrapper of the alert it would be the ideal usage. check this example:
https://mui.com/material-ui/react-snackbar/#customization

Reply:
@rnd500designs What about the controls, which component should be used to display the controls? at the moment I used Alert instead of Snackbar. If I'll change it to snackbar, then it will now have different argTypes/controls like anchorOrigin, onClose, open though the text inside the Alert will be hard coded instead of it coming from the args

```
  const meta: Meta<typeof MuiAlert> = {
   title: "Example/Banner Notification",
   component: MuiAlert,
   tags: ["autodocs"],
   argTypes: {
     children: {
       control: "text",
       description: "The content of the component."
     },
     // other args
    },
  }
 ```